_posts/2017-06-21-next-destination-zurich.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Next stop: Zürich!"
+socialImageSrc: 'A BIG FAT TODO RIGHT HERE'
+authors:
+  - andrew
+  - gandro
+---
+
+It's time to begin another adventure! After previous fun in Berlin and Kyiv, we're getting ready for Zürich, Switzerland! We hope you'll join us on **September 30th and October 1st, 2017** at [focusTerra](https://goo.gl/maps/cPY6GJoF8Hm) near ETH.
```

Comment:
It is! http://nominatim.openstreetmap.org/details.php?place_id=86029032

The address (which is unique for the building) is "Zürich, Sonneggstrasse 5", which is also on GMaps.